static/src/javascripts/projects/common/modules/commercial/epic-utils.js

```diff
@@ -35,59 +34,58 @@ const reportEpicError = (error: ReportedError): void => {
     reportError(error, { feature: 'epic' }, false);
 };
 
-const controlEpicComponent = (abTest?: ABTestVariant): EpicComponent => {
+const controlEpicComponent = (
+    abTest?: ABTestVariant
+): Promise<EpicComponent> => {
     const epicId = 'epic_control'; // TODO: check ok to use this
     const epicComponentType = 'ACQUISITIONS_EPIC';
-    let testimonialBlock = '';
-    if (epicControlCopy.testimonial) {
-        testimonialBlock = acquisitionsTestimonialBlockTemplate(
-            epicControlCopy.testimonial
-        );
-    }
-    const rawEpic = acquisitionsEpicControlTemplate({
-        copy: epicControlCopy,
-        componentName: '', // TODO: confirm data-component not needed
-        buttonTemplate: epicButtonsTemplate({
-            supportUrl: addTrackingCodesToUrl({
-                base: supportContributeURL,
-                componentType: epicComponentType,
-                componentId: epicId,
-                campaignCode: epicId,
-                abTest,
+    const testimonialBlock = '';
+
+    return epicControlCopy().then(copy => {
+        const rawEpic = acquisitionsEpicControlTemplate({
+            copy,
+            componentName: '', // TODO: confirm data-component not needed
+            buttonTemplate: epicButtonsTemplate({
+                supportUrl: addTrackingCodesToUrl({
+                    base: supportContributeURL,
+                    componentType: epicComponentType,
+                    componentId: epicId,
+                    campaignCode: epicId,
+                    abTest,
+                }),
             }),
-        }),
-        testimonialBlock,
-        epicClass: '',
-        wrapperClass: '',
-    });
-    const epic = $.create(rawEpic).get(0);
-    return {
-        html: epic,
-        componentEvent: {
-            component: {
-                componentType: epicComponentType,
-                id: epicId,
+            testimonialBlock,
+            epicClass: '',
+            wrapperClass: '',
+        });
+
+        const epicElement = $.create(rawEpic).get(0);
+        return {
+            html: epicElement,
+            componentEvent: {
+                component: {
+                    componentType: epicComponentType,
+                    id: epicId,
+                },
+                abTest,
             },
-            abTest,
-        },
-    };
+        };
+    });
 };
 
-const insertAtSubmeta = (epic: HTMLDivElement): Promise<void> =>
+const insertAtSubmeta = (epic: EpicComponent): Promise<EpicComponent> =>
```

Comment:
@Mullefa when you're back, you may see this and not like it - I appreciate the function signature is now less accurate. I did it just to make it easy for `displayControlEpic` to have the right return type (`insertAtSubmeta` would just pass through the `EpicComponent`) but you might find a nicer way. Also, I haven't really tested this code because it's not currently in use. (This is the downside of having two different libraries for displaying the epic). You may want to give it some scrutiny if you do decide to run another DFP test. Or indeed use these libs in any other test